doc/how-tos/load-data-into-pachyderm.md

```diff
@@ -0,0 +1,193 @@
+# Load Your Data Into Pachyderm
+
+Before you read this section, make sure that you are familiar with
+the [Data Concepts](../concepts/data-concepts/index.html) and
+[Pipeline Concepts](../concepts/pipeline-concepts/index.html).
+
+The data that you commit to Pachyderm is stored in an object store of your
+choice, such as Amazon S3, MinIO, Google Cloud Storage, or other. Pachyderm
+records the cryptographic hash (`SHA`) of each portion of your data and stores
```

Comment:
I was trying to fix two things with this paragraph:
* Remove the non-existing verb _content-address_ that was used in the original text.
* Give a little bit more detail about content-addressing in general, because I don't think a lot of people know what it is. We can remove it completely if you think it's too much.